data/strings/types_strings.txt

```diff
@@ -579,6 +581,7 @@
     ko = 바
     mr = बार
     nl = Bar
+    pl = Bar alkoholowy
```

Comment:
Are there non-alcoholic bars? :)

Some translations in types_strings.txt should be synchronized with data/categories.txt
Can you please double-check that they are equal? Otherwise, some object types may not be searchable.
See documentation at the top of data/categories.txt.

Reply:
> Are there non-alcoholic bars? :)

Yes. "bar" in Eglish and Polish are false friends.

See https://en.wikipedia.org/wiki/Bar_mleczny for start, but "bar" is in general fast food/low end restaurant.

See also https://en.wiktionary.org/wiki/bar#Polish

Reply:
> Some translations in types_strings.txt should be synchronized with data/categories.txt
Can you please double-check that they are equal? Otherwise, some object types may not be searchable.
See documentation at the top of data/categories.txt.

I guess that ideally it would be checked automatically, on PR opening?

Reply:
Thanks! That was interesting to learn. What about categories.txt sync?

Reply:
> What about categories.txt sync?

Maybe it would be at least possible to automatically detect mismatches?

Reply:
Please check related issues like https://github.com/organicmaps/organicmaps/issues/1841

Reply:
Already checked with tools/ruby/category_consistency/check_consistency.rb . No issues with Polish currently.